link.go

```diff
@@ -28,6 +28,14 @@ type LinkAttrs struct {
 	Namespace    interface{} // nil | NsPid | NsFd
 }
 
+// NewLinkAttrs returns LinkAttrs structure filled with default values
+func NewLinkAttrs() LinkAttrs {
+	return LinkAttrs{
+		TxQLen: -1,
+		MTU:    -1,
+	}
+}
+
```

Comment:
You're checking for `> 0` below so why not have "not set" be `== 0`. This way, default initialization still works and `NewLinkAttrs()` is not needed.